nodes/DictComp.py

```diff
@@ -1 +1,20 @@
+"""
+DictComp astroid node
+
+A DictComp node represents dictionary comprehensions in Python.
+
+Attributes:
+    - key         (node)
+        - The part that will be evaluated for each item.
```

Comment:
This description shouldn't be the same as the one for "value" - there is a difference that should be documented..